ruby-on-rails/ruby-on-rails-quiz.md

```diff
@@ -795,13 +796,13 @@ rescue => e
 end
 ```
 
-- [x] B
+- [ ] B
 
 ```ruby
 rescue_from ActiveRecord::RecordNotFound, with: :render_not_found_response
 ```
 
-- [ ] C
+- [x] C
```

Comment:
Apart from the Reference added that states the usage of cache.delete, I have not been able to find a single mention of `cache.destroy`.